test/integration/wan_lan_dht_test.go

```diff
@@ -51,8 +51,10 @@ func TestDHTConnectivitySlowRouting(t *testing.T) {
 }
 
 // wan prefix must have a real corresponding ASN for the peer diversity filter to work.
-var wanPrefix = net.ParseIP("2001:218:3004::")
-var lanPrefix = net.ParseIP("fe80::")
+var (
+	wanPrefix = net.ParseIP("2001:218:3004::")
+	lanPrefix = net.ParseIP("fe80::")
+)
```

Comment:
Could we turn this one off ? This is not helping with readability and make copy-pastability worst and spread the code thin.